web/src/utils/features-gate.ts

```diff
@@ -10,5 +9,5 @@ export enum Feature {
  */
 export const isAllowed = (f: Feature) => {
   const url = window.location.toString();
-  return url.includes('localhost:9000') || url.includes(`${f}=preview`);
+  return url.includes(`${f}=preview`);
```

Comment:
@jpinsonneau , as said yesterday I don't think we should have this localhost check for feature gates. As developers we don't necessarily want to get the in-progress stuff going around here and there, I think it's better to keep seeing something close to what users are seeing, and have feature gates still opt-in.